WordPress/Classes/ViewRelated/NUX/SiteCreationDomainsViewController.swift

```diff
@@ -8,7 +8,8 @@ class SiteCreationDomainsViewController: NUXViewController {
     @IBOutlet weak var buttonContainerViewBottomConstraint: NSLayoutConstraint!
     @IBOutlet weak var buttonContainerHeightConstraint: NSLayoutConstraint!
 
-    open var siteName: String?
+    // Used to store Site Creation user options.
+    var siteOptions: [String: Any]?
```

Comment:
If this inherited from a new class `SiteCreationViewController` instead of `NUXViewController` then `siteOptions` could be declared there once. Also, the part of `prepare()` that passes it on to the next VC could be defined in one spot. This is how `LoginViewController` is working atm.

Just a thought—there may be problems with it. Also, even if you like the idea it might fit in well with whenever you add `SiteCreationFields`.

Reply:
I do like the idea. It was my intent to resolve `siteOptions` as a whole when I added `siteCreationFields`, and adding a new class would fit nicely into that effort. Would you be OK with not doing that on this PR?

Reply:
Sure 😀